### DISCUSSION TITLE
How to install WebUI in mac M1.

### DISCUSSION THREAD

**Discussion by mesutatalik** [gitmylo/audio-webui#114] [Q&A]
Can someone guide me how to install everything? I don't really understand the code.
I tried to do the following .
1. I downloaded miniconda os . python 3.10
2. I go to the terminal and type: git clone https://github.com/gitmylo/audio-webui
something downloaded to my computer.
3. Then I go to the terminal and type again. ![Screenshot 2023-08-18 at 22 55 29](https://github.com/gitmylo/audio-webui/assets/137689095/fe87aa64-6bb8-467c-9589-bdc1b7bb7cae)
Then my computer reported an error zsh: command not found: run.sh .
I don't know what else to do.
I just typed the steps above and did nothing else.

**Comment by gitmylo** [comment]
1. It's recommended to use bash, not zsh
2. In bash, you can run the script with `./run.sh`, if you get a permissions denied error, run `chmod +x run.sh`
